weekly-updates/weekly-update.2015-05-29.md

```diff
@@ -0,0 +1,41 @@
+# io.js 2.2 releases
+This week we had two io.js releases [v2.2.0](https://iojs.org/dist/v2.2.0/) and [v2.2.1](https://iojs.org/dist/v2.2.1/), complete changelog can be found [on GitHub](https://github.com/iojs/io.js/blob/v1.x/CHANGELOG.md).
```

Comment:
Changelog URL is incorrect, it supposed to https://github.com/nodejs/io.js/blob/master/CHANGELOG.md